db/create_tables.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE peeps
```

Comment:
Useful to include the table setup SQL in the project.  These can be executed automatically by the travis.yml file for testing or a Rakefile (not sure how though...)